CHANGELOG.md

```diff
@@ -4,10 +4,60 @@ All notable changes to this project will be documented in this file.
 
 This projects adheres to [Semantic Versioning](http://semver.org/) and [Keep a CHANGELOG](http://keepachangelog.com/).
 
-## [Unreleased]
```

Comment:
Add a new section above:
```
[Unreleased]

_Nothing yet._
```

That then provides a link to compare the 0.10.0 against HEAD.